templates/_layout.html

```diff
@@ -0,0 +1,129 @@
+<!doctype html>
+<html class="no-js" lang="en" dir="ltr" itemscope itemtype="http://schema.org/Product">
+<head>
+
+  <title>{% block title %}{% endblock %} | Juju</title>
+
+  <meta charset="UTF-8" />
+  <meta name="description" content="{% block description %}{% endblock %}" />
+
+  <meta name="author" content="Canonical" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <!--[if IE]>
+  <meta http-equiv="X-UA-Compatible" content="IE=8">
+  <![endif]-->
+
+  <link rel="shortcut icon" href="/static/img/favicon.ico" type="image/x-icon" />
+
+  <link rel="apple-touch-icon-precomposed" sizes="57x57" href="/static/img/icons/apple-touch-icon-57x57-precomposed.png"/>
+  <link rel="apple-touch-icon-precomposed" sizes="60x60" href="/static/img/icons/apple-touch-icon-60x60-precomposed.png"/>
+  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="/static/img/icons/apple-touch-icon-72x72-precomposed.png"/>
+  <link rel="apple-touch-icon-precomposed" sizes="76x76" href="/static/img/icons/apple-touch-icon-76x76-precomposed.png"/>
+  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="/static/img/icons/apple-touch-icon-114x114-precomposed.png"/>
+  <link rel="apple-touch-icon-precomposed" sizes="120x120" href="/static/img/icons/apple-touch-icon-120x120-precomposed.png"/>
+  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/static/img/icons/apple-touch-icon-144x144-precomposed.png"/>
+  <link rel="apple-touch-icon-precomposed" sizes="152x152" href="/static/img/icons/apple-touch-icon-152x152-precomposed.png"/>
+  <link rel="apple-touch-icon-precomposed" sizes="180x180" href="/static/img/icons/apple-touch-icon-180x180-precomposed.png"/>
+  <link rel="apple-touch-icon-precomposed" href="/static/img/icons/apple-touch-icon-precomposed.png"/>
+
+
+  <!-- google fonts -->
+  <link href='https://fonts.googleapis.com/css?family=Ubuntu:400,300,300italic,400italic,700,700italic%7CUbuntu+Mono' rel='stylesheet' type='text/css' />
+
+  <!-- stylesheets -->
+  <link rel="stylesheet" type="text/css" media="screen" href="/static/css/styles.css" />
+
+  <script>window.app = {utils: {}};</script>
+
+  <meta name="twitter:card" content="summary">
+  <meta name="twitter:site" content="@ubuntucloud">
+  <meta name="twitter:creator" content="@ubuntucloud">
+  <meta name="twitter:domain" content="jujucharms.com">
+  <meta name="twitter:title" content="Jujucharms | Juju">
+  <meta name="twitter:description" content="Juju is an open source, application and service modelling tool from Canonical that helps you deploy, manage, and scale your applications on any cloud.">
+  <meta name="twitter:image" content="/static/img/juju-twitter.png">
```

Comment:
here you brought the metadata for juju
Update to landscape or set as empty for the tome being